apache-poi-3/src/test/java/com/baeldung/jexcel/JExcelDetectEmptyRowUnitTest.java

```diff
@@ -0,0 +1,44 @@
+package com.baeldung.jexcel;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import org.junit.Before;
+import org.junit.Test;
+
+import jxl.Cell;
+import jxl.Sheet;
+import jxl.Workbook;
+import jxl.read.biff.BiffException;
+
+
+public class JExcelDetectEmptyRowUnitTest {
+
+	private JExcelHelper jexcelHelper;
+	private static String FILE_NAME = "consumer_info_with_empty_row.xls";
+	private String fileLocation;
+
+	@Before
+	public void loadExcelFile() throws IOException {
+
+		fileLocation = "src/main/resources/" + FILE_NAME;
```

Comment:
This does not need to be initialized this way. `fileLocation` always has the same value and so should be `private static final String FILE_LOCATION` initialized in the class' static initializer list